process_noise:
  ownship:
    x: 0.0
    y: 0.0
    z: 0.001
    quat: 0.001 # Scaled by 100 in the code
    yaw: 0.01
    x_vel: 0.00001
    y_vel: 0.00001
    z_vel: 0.001
    yaw_vel: 0.001
    accel_bias: 0.0001 # m/s^3/sqrt(Hz), scaled by 3 in code
    gyro_bias: 0.000001 # m/s^3/sqrt(Hz), scaled by 7 in code
initial_uncertainty:
  ownship:
    x: 0.1
    y: 0.1
    z: 0.1
    quat: 1.0
    yaw: 1.0
    bias: 1.0
    x_vel: 0.3
    y_vel: 0.3
    z_vel: 0.3
    yaw_vel: 0.3
    